src/util/address.rs

```diff
@@ -870,6 +887,14 @@ impl fmt::Debug for Address {
     }
 }
 
+/// Convert a byte array of a pubkey hash into a segwit redeem hash
+pub fn segwit_redeem_hash(pubkey_hash: &[u8]) -> ::hashes::hash160::Hash {
```

Comment:
I don't think this should be public either. If yes, then maybe some other module.